sonar-ws/src/main/java/org/sonarqube/ws/client/users/UsersService.java

```diff
@@ -74,7 +74,7 @@ public CreateWsResponse create(CreateRequest request) {
         .setParam("login", request.getLogin())
         .setParam("name", request.getName())
         .setParam("password", request.getPassword())
-        .setParam("scmAccount", request.getScmAccount())
+        .setParam("scmAccount", request.getScmAccount() == null ? null : request.getScmAccount())
```

Comment:
What's the point of this change?

Reply:
no point, I badly fixed the generated code